releases/v1.1.0-rc.0.toml

```diff
@@ -0,0 +1,148 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
+Kubernetes [Container Runtime Interface](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md).
+CRI is a new plugin which allows connecting the containerd daemon directly to a
+Kubernetes Kubelet to be used as the container runtime. The CRI GRPC interface
```

Comment:
very minor, but not sure "Kubelet" needs capitalization here; I believe "kubelet" is fine